priv/gettext/en/LC_MESSAGES/errors.po

```diff
@@ -50,13 +50,23 @@ msgid "are still associated with this entry"
 msgstr ""
 
 ## From Ecto.Changeset.validate_length/3
+msgid "should have %{count} item(s)"
```

Comment:
are we actually _using_ `gettext` for translating the UI? 💭 
If so, we could add a whole new section to this tutorial! 💡